pkg/acs/calacs/calacs/calacs.c

```diff
@@ -203,51 +206,49 @@ int CalAcsRun (char *input, int printtime, int save_tmp, int verbose, int debug,
+    */
+ 
+    /* Loop over each entry in the association table to create a combined _spt.fits and a 
+       combined .tra file.  The terminology of "dth" is leftover from the original CALACS 
+       implementation.
+    */
     if (asn.process == FULL){
-        if (asn.verbose) {
-            trlmessage ("CALACS: Building DTH products");
-        }
         acsdth_input = NULL;
         for (prod = 0; prod < asn.numprod; prod++) {
-            /* Create empty DTH product, header only */
-            /* This uses only one sub-product for the header template,
-               but later versions should use a function similar to
-               BuildSumInput to create list of subproducts as inputs...
-            */
+
+            /* Create the list of input files */
             acsdth_input = BuildDthInput (&asn, prod);
 
             /* We always want to create a final concatenated trailer file for
-               the entire association whether there is a product or not. So, we
-               set up the trailer file based on the association file name itself.
+               the entire association.  Set up the trailer file based on the 
+               association file name itself.
             */
 
-            /* If desired, we could optionally use the full _drz.tra filename
-               as the trailer filename, based on the output dither product name.
-            if (strcmp(asn.product[prod].prodname,"") != 0) {
-                InitDthTrl (acsdth_input, asn.product[prod].prodname);
-            } else { */
-
+            trlmessage ("CALACS: Building combined TRA product");
             InitDthTrl(acsdth_input, asn.rootname);
 
-            /* End brace for optional dither product name assignment...
-            } */
+            /* Write out temp trailer file to final file */
+            WriteTrlFile ();
 
-            /* Check if we have a PROD-DTH specified...*/
+            /* Check if we have a PROD-DTH specified in the association table.  Even
+               though CALACS will not produce the drizzled product, the MEMTYPE of the 
+               rootname of the association dataset can be PROD-DTH.
+            */
             if (strcmp(asn.product[prod].prodname, "") != 0) {
 
                 if ((asn.dthcorr == PERFORM || asn.dthcorr == DUMMY)) {
+                    trlmessage ("CALACS: Building combined SPT product");
                     if (AcsDth (acsdth_input, asn.product[prod].prodname, asn.dthcorr, printtime, asn.verbose) )
                         return (status);
 
-                    /* Pass posid of 0 to indicate a PRODUCT is to be updated */
                     updateAsnTable(&asn, prod, NOPOSID);
                 }
             } else {
-                trlwarn ("No DTH product name specified. No product created.");
-                /*status = ACS_OK; */
+                trlwarn ("No indicator in the association table for combined SPT product to be generated.");
```

Comment:
Does "indicator" explicitly mean something or should this be less ambiguous?

Reply:
You got me as the term "indicator" is vague.  I was trying to avoid the use of "DTH" terminology which was one of several triggers for invoking AcsDth().   I could update the message to be "No combined product specified in the association table so no _spt.fits file to be generated.'